Controller/Payment/Order.php

```diff
@@ -36,13 +36,23 @@ public function __construct(
 
         $this->checkoutFactory = $checkoutFactory;
         $this->catalogSession = $catalogSession;
+	$this->config = $config;
     }
 
     public function execute()
     {
         $amount = (int) (round($this->getQuote()->getBaseGrandTotal(), 2) * 100);
 
         $receipt_id = $this->getQuote()->getId();
+	    
+	$payment_action = $this->config->getPaymentAction();
```

Comment:
indentation

Reply:
Indentation problem has been fixed. Kindly go ahead with the commit